.github/workflows/scanner-build.yaml

```diff
@@ -121,6 +133,12 @@ jobs:
         # Prevent fatal error "detected dubious ownership in repository" from recent git.
         git config --global --add safe.directory "$(pwd)"
 
+    - uses: ./.github/actions/handle-tagged-build
+
+    - name: Set build tag for prerelease images
+      if: matrix.name == 'prerelease'
+      run: echo "BUILD_TAG=$(make -C scanner --quiet --no-print-directory tag)-prerelease" >> "$GITHUB_ENV"
```

Comment:
I wonder if this could have been something like `echo "BUILD_TAG=${BUILD_TAG}-prerelease" >> "$GITHUB_ENV"`, since `BUILD_TAG` is already set through `handle-tagged-build` and we'd have one fewer places to update if the logic changes.

Reply:
probably, I just copied `build.yaml`, though, which did

```
echo "BUILD_TAG=$(make --quiet --no-print-directory tag)-prerelease" >> "$GITHUB_ENV"
```